fastlane-ios/README.md

```diff
@@ -27,8 +27,11 @@ This template:
    - Runs unit testing with [scan](http://docs.fastlane.tools/actions/run_tests/#whats-scan).
    - Performs static analysis on the codebase with [Swiftlint](https://github.com/realm/SwiftLint).
    - Builds the app with [gym](https://docs.fastlane.tools/actions/gym/).
+3. Adds any test failures as annotations using [junit-annotate-buildkite-plugin](https://buildkite.com/docs/agent/v3/cli-annotate#using-annotations-to-report-test-results).
```

Comment:
Should this link to the plugin? 

https://github.com/buildkite-plugins/junit-annotate-buildkite-plugin

Reply:
It probably should, yeah. Let me update the link!